client/js/connectionsAPI.js

```diff
@@ -19,8 +21,9 @@ var connectionsAPI = (function() {
 
   var rol = {
     endUser: 'End user',
-    restaurantViewer: 'Restaurant Viewer',
-    globalManager: 'global manager'
+    restaurantViewer: 'Restaurant viewer',
```

Comment:
It is specified in the FIWARE Tour Guide App - Functional Specification.
https://docs.google.com/document/d/1FpeuBzPkAkIs90rP16jeOdi5-JuSbcdsiOatE87l3Dw/edit

 It is also returned from the implemented API. For example user1 has the roles:
```
"roles":[{"name":"End user","id":"f715a71095c447e7bd5bccb1e65d4bf5"}]
```